__tests__/reactTags.test.js

```diff
@@ -562,10 +568,10 @@ describe('Test ReactTags', () => {
         })
       );
       const $input = $el.find('.ReactTags__tagInputField');
-      $input.simulate('change', { target: { value: 'Ea' } });
+      $input.simulate('change', { target: { value: 'Or' } });
```

Comment:
'Or' still here because test message 'should add tag with custom label field and default suggestion filter' mean there have to be query with result of zero suggestions to add new tag, as i understand.